locale/en/messages.json

```diff
@@ -84,7 +87,6 @@
   "An error has occurred": "",
   "An error has occurred.": "",
   "An unknown error occurred": "",
-  "Anonymous Bind": "",
```

Comment:
Anonymous Bind, Bind Type, LDAP Credentials should not be removed here. Is your plugins-private repo updated?